Task7FinalReact/valery-youtube-searcher/src/App.module.css

```diff
@@ -0,0 +1,27 @@
+/*Обнуление*/
```

Comment:
I think you should resetting all global styles in root `index.css`.

It's better to use ready packages like `reset.css` for resetting all browser-dependent properties.